doc_conf/conf.py

```diff
@@ -51,172 +37,112 @@
 html_theme_options = {
-    "navbar_sidebarrel": False,
-    "navbar_pagenav": False,
-    "source_link_position": "",
-    "navbar_links": [
-        ("Examples", "auto_examples/index"),
-        ("API", "api"),
-        ("GitHub", "https://github.com/mind-inria/hidimstat", True),
+    "icon_links": [
+        {
+            "name": "GitHub",
+            "url": "https://github.com/mind-inria/hidimstat/",
+            "icon": "fa-brands fa-github",
+        },
     ],
-    "bootswatch_theme": "flatly",
-    "bootstrap_version": "3",
+    "navbar_end": ["version-switcher", "theme-switcher", "navbar-icon-links"],
+    "header_links_before_dropdown": 4,
 }
 
-# Add any paths that contain custom themes here, relative to this directory.
-html_theme_path = sphinx_bootstrap_theme.get_html_theme_path()
-
-# Add any paths that contain custom static files (such as style sheets) here,
-# relative to this directory. They are copied after the builtin static files,
-# so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ["_static"]
-
-
-# -- Options for HTMLHelp output ------------------------------------------
-
-# Output file base name for HTML help builder.
-htmlhelp_basename = "hidimstat_doc"
 
-
-# -- Options for LaTeX output ---------------------------------------------
-
-latex_elements = {
-    # The paper size ('letterpaper' or 'a4paper').
-    #
-    # 'papersize': 'letterpaper',
-    # The font size ('10pt', '11pt' or '12pt').
-    #
-    # 'pointsize': '10pt',
-    # Additional stuff for the LaTeX preamble.
-    #
-    # 'preamble': '',
-    # Latex figure (float) alignment
-    #
-    # 'figure_align': 'htbp',
+# -- Options for gallery ----------------------------------------------------
+# Generate the plots for the gallery
+matplotlib.use("agg")
+sphinx_gallery_conf = {
+    "examples_dirs": ["../examples"],
+    "gallery_dirs": ["auto_examples"],
+    "plot_gallery": "True",
+    "thumbnail_size": (160, 112),
+    "min_reported_time": 1.0,
+    "abort_on_example_error": False,
+    "image_scrapers": ("matplotlib",),
+    "show_memory": True,
+    "doc_module": "hidimstat",
+    "backreferences_dir": "generated",
+    "reference_url": {
+        # The module we locally document (so, hidimstat) uses None
+        "hidimstat": None,
+        # We don't specify the other modules as we use the intershpinx ext.
+        # See https://sphinx-gallery.github.io/stable/configuration.html#link-to-documentation  # noqa
+    },
 }
 
-# Grouping the document tree into LaTeX files. List of tuples
-# (source start file, target name, title,
-#  author, documentclass [howto, manual, or own class]).
-latex_documents = [
-    (
-        master_doc,
-        "hidimstat.tex",
-        "hidimstat Documentation",
-        "Jerome-Alexis Chevalier",
-        "manual",
-    ),
-]
-
-
-# -- Options for manual page output ---------------------------------------
+git_root_url = "https://github.com/mind-inria/hidimstat"
```

Comment:
```suggestion
```

Reply:
Move in the section for describing the project.

Reply:
done